client/components/payment-activity/index.tsx

```diff
@@ -5,23 +5,52 @@ import * as React from 'react';
 import { Card, CardBody, CardHeader } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import interpolateComponents from '@automattic/interpolate-components';
+import moment from 'moment';
 
 /**
  * Internal dependencies
  */
 
 import EmptyStateAsset from 'assets/images/payment-activity-empty-state.svg?asset';
-import PaymentActivityData from './payment-activity-data';
+import PaymentActivityDataComponent from './payment-activity-data';
```

Comment:
Kudos for clarifying this component and fixing the naming conflict with the TS interface.